robottelo/ssh.py

```diff
@@ -241,6 +241,22 @@ def upload_file(local_file, remote_file, key_filename=None, hostname=None):
             sftp.close()
 
 
+def upload_files(local_dir, remote_dir, file_type='.txt'):
```

Comment:
to be more general perhaps instead of file_type use a more general file_match='*.txt' and use fnmatch function to check file file name
what do you think?

Reply:
Nice one, that's even better

Reply:
Yes, that is more clean